vocabularies/UI.json

```diff
@@ -1327,6 +1327,18 @@
                 "@Core.Description": "Value for the key property"
             }
         },
+        "ExpressionParameter": {
```

Comment:
You don't need this, just use the concrete type [`Parameter`](https://github.com/SAP/odata-vocabularies/blob/master/vocabularies/UI.md#Parameter) and use an expression for `PropertyValue`.

That way it is clear what the expression language is: EDM annotation expressions.